external/storm-solr/src/main/java/org/apache/storm/solr/mapper/SolrMapper.java

```diff
@@ -26,6 +26,7 @@
 import java.util.List;
 
 public interface SolrMapper extends Serializable {
+    void configure();
```

Comment:
@satishd  Thanks for the review.  Since we are using  java 1.7 compilation for 1.x branch, we can not use default methods. We can do it on mater branch. I don't see any option other than breaking the compatibility here. Let me know your thoughts.

Updated PR with review comments

Reply:
Is there no chance to add it to some kind of configurations (with proper default) and move out of interface? Just to find alternatives which don't break backward compatibility.

Reply:
Looks like major change is RestJsonSchemaBuilder. RestJsonSchemaBuilderV1 builds (fetches) schema in constructor, whereas RestJsonSchemaBuilderV2 builds (fetches) schema in `buildSchema()` which is newly added to interface in SchemaBuilder as well. So the implementation is going to require explicit initialization.

How about separating this into two pull requests which for 1.x-branch is initializing schema in constructor as same as what RestJsonSchemaBuilderV1 is doing (so no need to add `configure()` and `buildSchema()`), and which for master is same as current PR?

Reply:
@HeartSaVioR Yes, thats a reasonable proposal. On 1.x-branch, We can document that we don't support security with RestJsonSchemaBuilder

Reply:
@satishd Could you put opinion with my approach, or any new alternatives?

Reply:
@HeartSaVioR  Suggested approach looks practical to me. If there are requests for this issue to be fixed in 1.x then new interfaces can be introduced without changing existing interfaces and make respective code changes which support both old and new semantics.